ui/packages/catalog-ui-search/src/main/webapp/component/dropdown/query-src/dropdown.query-src.view.js

```diff
@@ -17,13 +17,65 @@ const Marionette = require('marionette')
 const _ = require('underscore')
 const $ = require('jquery')
 const DropdownView = require('../dropdown.view')
-const template = require('./dropdown.query-src.hbs')
 const ComponentView = require('component/query-src/query-src.view')
 const sources = require('component/singletons/sources-instance')
 const properties = require('properties')
+import * as React from 'react'
+import LocalizationContext, {
+  Context,
+} from '../../../react-component/container/localization'
 
 module.exports = DropdownView.extend({
-  template: template,
+  template(props) {
+    return (
+      <div>
+        <div className="dropdown-label">
+          <LocalizationContext.Consumer>
+            {(context: Context) => {
+              return context.keywords['Sources']
+            }}
+          </LocalizationContext.Consumer>
+        </div>
+        <div className="dropdown-container">
+          <div className="dropdown-text is-input">
+            {props.enterprise ? (
+              <span className="text-src is-available">
+                <LocalizationContext.Consumer>
+                  {(context: Context) => {
+                    return 'All ' + context.keywords['Sources']
+                  }}
+                </LocalizationContext.Consumer>
+              </span>
+            ) : (
+              props.sources.map(source => (
+                <span
+                  className={
+                    'text-src ' + (source.available ? 'is-available' : null)
```

Comment:
✏️ Null should be changed to `''`. Otherwise the className `null` will get added

Reply:
Yeah I was kinda curious about that when I was writing this. I had hoped react would be smarter